Firmware/mmu2_reporting.cpp

```diff
@@ -252,13 +253,7 @@ void ReportErrorHook(CommandInProgress /*cip*/, uint16_t ec, uint8_t /*es*/) {
                 ReportErrorHookState = ReportErrorHookStates::RENDER_ERROR_SCREEN;
                 break;
             case 2:
-                // Exit error screen and enable lcd updates
-                lcd_set_custom_characters();
-                lcd_update_enable(true);
-                lcd_return_to_status();
-                // Reset the state in case a new error is reported
-                mmu2.is_mmu_error_monitor_active = false;
-                ReportErrorHookState = ReportErrorHookStates::RENDER_ERROR_SCREEN;
+                ReportErrorHookState = ReportErrorHookStates::DISMISS_ERROR_SCREEN;
```

Comment:
How is this change related to sounds?

Reply:
I would have needed to add a call to `sound_wait_for_user_reset();` here. When I wrote the error screen code this change above didn't save memory, but for some reason now it saves 10 bytes.